.github/workflows/ui-rm_head_2.7.yaml

```diff
@@ -1,5 +1,5 @@
 # This workflow calls the master E2E workflow with custom variables
-name: UI-RM_head_2.7
+name: UI-RM_head_upgrade
```

Comment:
ditto here and on the rest of the document, including the name of the yaml file

Reply:
This 2.7 YAML is just to run upgrade job with my changes, before merge I am going to revert back to original state. See last commit message for it.